client/js/components/MapView/MapView.js

```diff
@@ -80,7 +80,8 @@ class MapViewComponent extends Component {
                           latitude: e.nativeEvent.coordinate.latitude,
                           longitude: e.nativeEvent.coordinate.longitude
                         }
-                      });
+                      }),
+                        this.props.navigation.navigate("AddLocker", {});
```

Comment:
why are you passing an empty object with this navigation?

Reply:
Now passing lats and longs.